app/services/courses/completed_webhook_data_service.rb

```diff
@@ -0,0 +1,17 @@
+module Courses
+  class CompletedWebhookDataService
```

Comment:
Something on the lines of `CompletedStatus` could be better I guess.

Reply:
I'd use the word _Completion_ here. 😅

Reply:
Done